packages/perseus/src/widgets/reaction-diagram.tsx

```diff
@@ -118,13 +116,13 @@ class ReactionDiagramWidget extends React.Component<any> {
                     return (
                         <div key={id} className="molecule-container">
                             <Molecule
-                                id={id}
+                                widgetId={id}
                                 rotationAngle={this.props.rotationAngle[i]}
                                 smiles={s}
                             />
                             {i === this.props.smiles.length - 1 ? null : (
                                 <Separator
-                                    data={this.props.separators[i]}
+                                    data={this.props.separators[i].data}
```

Comment:
Another place where logic changes I think. Are we sure it's safe? I realize these are deprecated widgets, but I think they're still in use. Although you might be fixing the bugs that made them problematic for all I know 😂 Just want to make sure we know we're not breaking prod.